src/adagrams.js

```diff
@@ -1,8 +1,97 @@
+const poolOfLetters = ( 
```

Comment:
It would be best to include each of these objects `poolOfLetters` and `valueHash` inside the Adagrams module (I know this is different then what I said last week :)